content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,79 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](/docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic management
+
+- **Added** [automatic determination](/docs/ops/traffic-management/protocol-selection/) of protocol types. You no longer need to explicitly prefix service ports with their protocol type.
+- **Added** a mode to the Gateway API for mutual TLS operation.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** Envoy proxy readiness checks. They now check Envoy's readiness status.
+- **Improved** container ports are no longer required in the pod spec. All ports are [captured by default](/faq/traffic-management/#controlling-inbound-ports).
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](/faq/traffic-management/#controlling-inbound-ports) by default.
+- **Improved** Pilot by reducing CPU utilization. The reduction approaches 90% depending on the specific deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecar` resource.
```

Comment:
```suggestion
- **Added** trust domain validation for services using mutual TLS. By default, the server only authenticates the requests from the same trust domain.
```

Reply:
This sentence is still not correct. I revised it.

Reply:
link to docs ?

Reply:
accepted.